src/components/ViolationForm.js

```diff
@@ -105,52 +123,27 @@ export const ViolationForm = () => {
         />
         <div className="form-control">
           <label className="inputLabel">Име</label>
-          <input
-            type="text"
-            name="name"
-            {...register('name', { required: true })}
-          />
-          {errors.name && errors.name.type === 'required' && (
-            <p className="errorMsg">Полето е задължително.</p>
-          )}
+          <input type="text" name="name" {...register('name')} />
+          <p className="errorMsg">{errors.name?.message}</p>
         </div>
         <div className="form-control">
           <label className="inputLabel">Имейл</label>
-          <input
-            type="email"
-            name="email"
-            {...register('email', {
-              required: true,
-            })}
-          />
-          {errors.email && errors.email.type === 'required' && (
-            <p className="errorMsg">Полето е задължително.</p>
-          )}
-          {errors.email && errors.email.message && (
-            <p className="errorMsg">{errors.email.message}</p>
-          )}
+          <input type="email" name="email" {...register('email')} />
+          <p className="errorMsg">{errors.email?.message}</p>
         </div>
         <div className="form-control">
           <label className="inputLabel">Телефон</label>
-          <input
-            type="text"
-            name="phoneNumber"
-            {...register('phoneNumber', { required: true })}
-          />
-          {errors.phoneNumber && errors.phoneNumber.type === 'required' && (
-            <p className="errorMsg">Полето е задължително.</p>
-          )}
+          <input type="text" name="phoneNumber" {...register('phoneNumber')} />
+          <p className="errorMsg">{errors.phoneNumber?.message}</p>
         </div>
         <div className="form-control">
           <label className="inputLabel">Описание на нарушението</label>
           <textarea
             id="violationText"
             name="description"
-            {...register('description', { required: true })}
+            {...register('description')}
           />
-          {errors.violationText && errors.violationText.type === 'required' && (
-            <p className="errorMsg">Полето е задължително.</p>
-          )}
+          <p className="errorMsg">{errors.description?.message}</p>
```

Comment:
```suggestion
					{errors.description &&
          	<p className="errorMsg">{errors.description.message}</p>}
```
Нека променим и другите съобщения така, че да не добавяме празни параграфи, когато няма грешки.